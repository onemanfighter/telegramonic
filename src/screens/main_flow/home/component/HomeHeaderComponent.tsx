import { Box, Text } from '@chakra-ui/react';
import { useTranslation } from 'react-i18next';

const HomeHeaderComponent = () => {
  const { t } = useTranslation();
  return (
    <Box width={'100%'}>
      <Box
        flex={1}
        flexDirection={'column'}
        justifyContent={'center'}
        border={'1px solid #00000035'}
        borderRadius={'lg'}
        margin={2}
        padding={2}
      >
        <Text fontSize={'x-large'} fontWeight={'600'} textAlign={'center'}>
          {t('Home.header.title')}
        </Text>
        <Text fontSize={'large'} fontWeight={'400'} textAlign={'center'}>
          {t('Home.header.subTitle')}
        </Text>
      </Box>
    </Box>
  );
};

export default HomeHeaderComponent;
