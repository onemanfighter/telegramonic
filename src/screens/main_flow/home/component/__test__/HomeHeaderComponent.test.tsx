import { render } from '@testing-library/react';
import HomeHeaderComponent from '../HomeHeaderComponent';

describe('HomeHeaderComponent', () => {
  it('should render correctly', () => {
    const { container } = render(<HomeHeaderComponent />);

    expect(container).toMatchSnapshot();
  });
});
