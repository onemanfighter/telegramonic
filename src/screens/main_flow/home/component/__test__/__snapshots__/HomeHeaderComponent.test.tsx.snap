// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeHeaderComponent should render correctly 1`] = `
<div>
  <div
    class="css-8atqhb"
  >
    <div
      class="css-77af5c"
    >
      <p
        class="chakra-text css-x3elb1"
      >
        Home.header.title
      </p>
      <p
        class="chakra-text css-1uf6a8x"
      >
        Home.header.subTitle
      </p>
    </div>
  </div>
</div>
`;
